### ISSUE TITLE
nLines not working

### ISSUE THREAD

**Issue by wesleymv** [lucagrulla/node-tail#140]
I am trying to use the nLines option, but it does not seem to work. When the file is modified, it brings more than one line.

Here is the code I am using:

```
Tail = require('tail').Tail;

tail = new Tail("C:/Users/pc/Desktop/watchfile/1.log", {nLines: 1});

tail.on("line", function(data) {
  console.log(data);
});

tail.on("error", function(error) {
  console.log('ERROR: ', error);
});
```

When it starts, it brings one line. But, brings many lines when file is modified.

I am sorry if this is just the wrong use of the library.

Thank you in advance

**Comment by lucagrulla** [comment]
Hello @wesleymv 

Are the multiple lines tailed when the file changes new lines?

**Comment by wesleymv** [comment]
Hi @lucagrulla 

The lines I get are in the old file. For example, 1.log contains:
`1
2
3`

when I add 4, this is the output:

`31
2
3`

if I remove the nLines and run the same steps (seems to omit last line):

`1
2
3`

**Comment by lucagrulla** [comment]
How are lines added?

The underline NodeJS core library is sometimes platform dependent. Are the
new lines added via an app, like notepad, or with a different mechanism?

**Comment by lucagrulla** [comment]
Fixed with v2.2.4
